multi-agent/lib/mas/resources/service.py

```
@@ -15,40 +17,49 @@
         self.element_registry = element_registry
         self._card_service = card_service
         self._dependency_resolver = DependencyResolver(resource_registry=self._store)
         self._validation_service = validation_service
         self._auth_service = auth_service
-
-        from global_utils.utils.crypto import FieldCipher
-        self._cipher = FieldCipher(encryption_key) if encryption_key else None
+        self._fields = field_encryption
 
     # ---------- CRUD ----------
     def create(self, *, identity: Identity, category, type, name, config) -> Resource:
```

Ruff (0.16.1)

[error] Function argument `type` is shadowing a Python builtin

(A002)